services/visual_recognition/v3.js

```diff
@@ -155,6 +108,12 @@ module.exports = function (RED) {
       apikey = sAPIKey || this.credentials.apikey;
       this.status({}); 
 
+      var visualRecognition = watson.visual_recognition({
+          api_key: apikey,
+          version: 'v3',
+          version_date: '2016-05-19'
+        });
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 6 space characters but found 8.](https://www.codacy.com/app/BetaWorks-NodeRED-Watson/node-red-node-watson/file/2713632089/issues/source?bid=3330694&fileBranchId=3344618#l115)